packages/babel-plugin-transform-computed-properties/test/fixtures/assumption-setComputedProperties/accessors/output.js

```diff
@@ -1,10 +1,10 @@
-var _foobar, _foobar2, _test, _test2, _obj, _mutatorMap;
-var obj = (_obj = {}, _foobar = foobar, _mutatorMap = {}, _mutatorMap[_foobar] = _mutatorMap[_foobar] || {}, _mutatorMap[_foobar].get = function () {
+var _obj;
+var obj = (_obj = {}, babelHelpers.defineAccessor(_obj, foobar, "get", function () {
```

Comment:
This is very nitpicky, the PR is very good as it is.

However, `babelHelpers.defineAccessor("get", _obj, foobar, function () {` gzips better, because gzip can deduplicate all the `defineAccessor("get",` parts in the code to refer to the same byte sequence. With `defineAccessor(_obj, foobar, "get"`, the only constant part that gzip can easily deduplicate is only `defineAccessor(`, because the `_obj` variable will likely have a different name for different calls.

Reply:
Done in 1fd56a3c4e900159ae31202c5aae8e90c88042c6.